packages/main/src/run-main.ts

```diff
@@ -49,16 +49,20 @@ async function runHelper(): Promise<void> {
     runNameSpec,
     gitSourceSpec
   )
-  core.setOutput(
+  tl.setVariable(
     DATABRICKS_RUN_NOTEBOOK_OUTPUT_KEY,
-    runOutput.notebookOutput.result
+    runOutput.notebookOutput.result,
+    false,
```

Comment:
Based on implementation we are specifying `secret: false, isOutput: true`


https://github.com/microsoft/azure-pipelines-task-lib/blob/master/node/task.ts#L163